contrib/examples/apiserver/instance.yaml

```diff
@@ -4,5 +4,5 @@ metadata:
   name: test-instance
   namespace: test-ns
 spec:
-  externalServiceClassName: test-serviceclass
-  externalServicePlanName: example-plan-1
+  externalClusterServiceClassName: test-serviceclass
```

Comment:
What end-goal UX are we aiming for here once we also have namespaced classes? Will we have two fields, `externalClusterServiceClassName` and `externalServiceClassName`, and the user will fill in one of them, and our instance will automatically link to the correct resource?

Reply:
During some free time on one of our calls I would love to hear more about why a resource that can be both namespaced or not was so problematic.

Reply:
That is my understanding on conversations with @pmorie, etc.